.github/workflows/build.yml

```diff
@@ -75,10 +81,13 @@ jobs:
             sudo -u postgres psql -c "CREATE EXTENSION postgis" npgsql_tests
           fi
 
-          export PGDATA=/etc/postgresql/${{ matrix.pg_major }}/main
           sudo sed -i 's/#ssl = off/ssl = on/' $PGDATA/postgresql.conf
           sudo sed -i 's/#max_prepared_transactions = 0/max_prepared_transactions = 10/' $PGDATA/postgresql.conf
-          sudo sed -i 's/#password_encryption = md5/password_encryption = scram-sha-256/' $PGDATA/postgresql.conf
+          if [ ${{ matrix.pg_major }} -ge 14 ]; then
+            sudo sed -i 's/password_encryption = md5/password_encryption = scram-sha-256/' $PGDATA/postgresql.conf
```

Comment:
So above we set to MD5, and now back to SCRAM (also SCRAM for pre-PG14)? Any reason this needs to be done?

Reply:
The idea was to create the default user with md5 encrypted password and the scram user with scram encrypted password. And I encountered some really weird problems without this 'dance'...

Reply:
OK @vonzshik, am OK with merging as-is.